test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -1079,6 +1079,21 @@ TEST_P(ExtProcIntegrationTest, GetAndRespondImmediately) {
   EXPECT_EQ("{\"reason\": \"Not authorized\"}", response->body());
 }
 
+TEST_P(ExtProcIntegrationTest, GetAndRespondImmediatelyWithNullCharacter) {
```

Comment:
The test name implies null character, but it is actually '\n'

Reply:
done